src/functionalTest/resources/features/F-002/S-021.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "must receive an error response for a non existing document id",
+  "_guid_": "S-021",
+  "_extends_": "F-002_Test_Data_Base",
+
+  "specs": [
+    "contains a non existing document id"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "documentId": "00000000-0000-0000-0000-000000000000"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
```

Comment:
these may also be inherited. pls double-check.

Reply:
I could not find these headers in following common headers:
Common_404_Response
Common_Response_Headers